redwood-dom-testing/src/commonMain/kotlin/app/cash/redwood/dom/testing/Frame.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2025 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.dom.testing
+
+/**
+ * Configure the canvas that we will snapshot our element on.
+ */
+public class Frame(
```

Comment:
Hmm I don't think this should live here. Or at least should immediately move after merging.

I think this should live in the shared test infrastructure so that each abstract test class can define their default frame (or even per-snapshot frames). This results in each platform producing images that are the same size and (should) look the same.

Reply:
Fantastic idea. Will do in follow-up.

Reply:
https://github.com/cashapp/redwood/issues/2806